pages/api/search.tsx

```diff
@@ -44,9 +44,7 @@ const excludedRepos = [
   "replayio/customer-support",
 ];
 
-const URL = true
-  ? "https://hasreplay.graphcdn.app"
-  : "https://api.github.com/graphql";
+const URL = "https://api.github.com/graphql";
```

Comment:
haha yeah - graphcdn rebranded as stellate and i haven't figured out how to fix the issue we're seeing there. Probably 5 mins too... 😂

Reply:
`¯\_(ツ)_/¯`

Switching to ISR for this case removes the need for Stellate IMO because the static site is still cached at the edge and requests to the GraphQL API don't block rendering.

Reply:
True